### DISCUSSION TITLE
Automatic configuration of client

### DISCUSSION THREAD

**Discussion by rux2** [rustdesk/rustdesk#6189] [Q&A]
Hi everyone, i've installed rustdesk on debian virtual machine. Now i would like to configure client directly from the server so that users don't insert manually server settings. I've read docs, but i wasn't able to do it.
Thanks for helping.

**Comment by noelhibbard** [comment]
In the official docs there is a sample Bash deploy script that you can use as a starting point:
https://rustdesk.com/docs/en/self-host/client-deployment/#linux

I haven't looked over the Bash script but I know their sample PowerShell deploy script doesn't work but it didn't take much effort to modify it so it does work.

**Reply by rux2** [reply]
Thanks for replying, since i'm new to this type of installations, can you give me some advice on what should i modify?
